doc/data/messages/r/redundant-unittest-assert/good.py

```diff
@@ -3,5 +3,9 @@
 
 class DummyTestCase(unittest.TestCase):
     def test_dummy(self):
-        actual = "test_result"
-        self.assertEqual(actual, "expected")
+        # Nothing, as an assert of a string literal will always pass
+        pass
+
+def test_division():
+    a = 9 / 3
+    assert a == 3
```

Comment:
If we do `# nothing` above we should also do so here? Note, I actually think we should keep the code instead of the comment.